clouddriver-titus/src/main/groovy/com/netflix/spinnaker/config/TitusConfiguration.groovy

```diff
@@ -63,7 +63,9 @@ class TitusConfiguration {
     List<NetflixTitusCredentials> accounts = new ArrayList<>()
     for (TitusCredentialsConfig.Account account in titusCredentialsConfig.accounts) {
       List<TitusRegion> regions = account.regions.collect {
-        new TitusRegion(it.name, account.name, it.endpoint, it.autoscalingEnabled, it.loadBalancingEnabled, it.applicationName, it.url, it.port, it.featureFlags)
+        String eurekaName = it.eurekaName == null ? null : it.eurekaName
+        String eurekaRegion = it.eurekaRegion == null ? null: it.eurekaRegion
```

Comment:
It seems like this should just be passed in? Not following the null check if you are setting it to null.

Reply:
yes fixed it , it was the related to the other comment , had defaults set initially and did not clean up after.